index.js

```diff
@@ -86,6 +86,17 @@ const sendResponse = ( req, settings, fn ) => {
 };
 
 /**
+ * Returns `true` if `v` is a File Form Data, `false` otherwise.
+ *
+ * @param {Mixed} v - instance to analize
+ * @return {Boolean} `true` if `v` is a DOM File instance
+ * @private
+ */
+function isFile( v ) {
+	return v instanceof Object && v.fileContents instanceof Blob;
```

Comment:
I don't know, I was thinking the same thing, but when I dumped the formData on the proxy-request handler, it was there too